ArmVirtPkg/ArmVirtPkg.ci.yaml

```diff
@@ -56,7 +56,8 @@
             "ArmPlatformPkg/ArmPlatformPkg.dec",
             "PcAtChipsetPkg/PcAtChipsetPkg.dec",
             "SecurityPkg/SecurityPkg.dec",
-            "ShellPkg/ShellPkg.dec"  #Is this ok?
+            "ShellPkg/ShellPkg.dec",  #Is this ok?
+            "QemuQ35Pkg/QemuQ35Pkg.dec"  #This probably not ok?

```

Comment:
i think this points at the need for common virtual platform package.

Reply:
That seems right.. But it will eventually be as rich as the OvmfPkg. Also I think this package's hierarchy is off.